index.html

```diff
@@ -4,17 +4,16 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Poppins">
     <link rel="stylesheet" href="css/style.css">
     <title>Porfolio Project</title>
 </head>
 <body>
+    <header>
     <nav>
-        <a class="logo" href="">My Logo</a>
+        <a class="logo" href="#">Joshua Blue-Jack</a>
         <ul  class="navbar">
-            <li>
-                <div class="humburger"> <a href="#"><img class="toggler-icon" src="img/hamburger-icon.png" alt="Hamburger Icon"></a></div>
-            <div class="nav-options">
+            
+            <div class="nav-options" id="myoverlay">
                 <ul class="navbar-nav">
                     <li class="nav-item">
                         <a class="nav-link" href="#portfolio">Portfolio</a>
```

Comment:
- [ ] Kindly follow the design more closely, move the options up and the `X` is looking a bit deformed , check the picture for more info
Design:
![image](https://user-images.githubusercontent.com/62705947/195057911-7b2ff50c-6315-4b8d-9b71-599f66a33040.png)

Yours:
![image](https://user-images.githubusercontent.com/62705947/195058027-8b03cf73-cbc4-46f0-9d92-52201ec3e1d0.png)